feature/appdetail/src/main/java/com/merxury/blocker/feature/appdetail/AppDetailViewModel.kt

```diff
@@ -708,16 +712,27 @@ class AppDetailViewModel @Inject constructor(
         } else {
             val packageInfo = pm.getPackageInfoCompat(packageName, 0)
             _appInfoUiState.emit(
-                AppInfoUiState.Success(app.toAppItem(packageInfo = packageInfo)),
+                AppInfoUiState.Success(
+                    app.toAppItem(packageInfo = packageInfo),
+                    getAppIcon(packageInfo),
+                ),
             )
         }
     }
+
+    private fun getAppIcon(packageInfo: PackageInfo?): Bitmap? {
+        val icon: Drawable? = packageInfo?.applicationInfo?.loadIcon(pm)
```

Comment:
This is a time-consuming function, please dispatch it to the worker thread, or use AppIconLoader to load the bitmap.